### DISCUSSION TITLE
[Projects beta] Kanban WIP limits

### DISCUSSION THREAD

**Discussion by ErisDS** [community/community#7459] [Projects and Issues]
This is a super advanced request given where the feature is, but wanted to see if I'm alone.

One thing I really miss from projects OG and beta, and that also is not available in more focused trackers like Linear, is proper Kanban WIP limits. 

I want to be able to set limits for how many items are allowed to be in progress, and how long they can stay there and then have GitHub projects reflect that in the UI. The most github-compatible way would be with labels, buuuut I'd love to see the issue be highlighted in a different color.

ATM there's zero visibility on when e.g. a status field changed, so I'm concerned it won't even be possible to implement this manually with workflows.

**Comment by curtisbarnard** [comment]
I agree, it looks like this feature request has been brought up a few times. Hopefully it's in the works to be added.

**Comment by darpank** [comment]
More interest for this feature is also at https://github.com/community/community/discussions/4848

**Comment by nbarbettini** [comment]
Simply highlighting a column in red if contains more than a fixed amount of issues would be useful.

**Comment by redcatbear** [comment]
Since we had the same need, we created a browser Plugin, that adds WiP limits. That does the trick for us until GitHub adds the feature.

At the moment the plugin is only tested with Firefox. You can find the plugin package on https://addons.mozilla.org/en-US/firefox/addon/github-project-tweaks/

It's Open Source, feedback and contributions welcome: https://github.com/exasol/github-project-tweaks-browser-extension

![WiP limit screenshot](https://user-images.githubusercontent.com/584162/228735928-ca600216-c22b-4655-b406-6dddbe0f0c17.png)

**Comment by probitcarwyn** [comment]
Column limits have been added, but they are not enforced, they are purely informational. It would be good to be able to enforce these limits.

I'll open another discussion for this, but in Kanban it's also important to be able to set a limit across a collection of columns, not just one.

**Reply by darpank** [reply]
They used to turn red for us, the recent changes with the Increased Items Beta might have broken this. I have reported it here - https://github.com/orgs/community/discussions/139936#discussioncomment-10830146 and it is acknowledged as a bug.

@probitcarwyn  If you have a minute, please add your support for the fix to that thread, and thanks.

**Reply by probitcarwyn** [reply]
Now that you mention it, yes they have stopped going red. Ideally I'd like to have the option to block adding issues,
